contract_compassion/view/end_contract_wizard_view.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2014 Compassion (http://www.compassion.ch)
+    @author Emanuel Cino <ecino@compassion.ch>
+    The licence is in the file __openerp__.py
+-->
+<openerp>
+    <data>
+		<record id="end_contract_view" model="ir.ui.view">

```

Comment:
Fix indentation of file (in notepad++ -> Completments -> Xml tools -> Pretty print